src/figuresClasses.ts

```diff
@@ -1,11 +1,104 @@
-export interface Figure {}
+enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
+
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: Color;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle';
```

Comment:
The `shape` property should be declared as a string literal type `'triangle'` to match the `Figure` interface. However, TypeScript allows this syntax, so it's not a critical issue.